docs/pages/blog/premium-plan-release.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Premium passengers, please proceed to the boarding gate 🚀
+description: Introducing the MUI X Premium plan, and a new licensing model
+date: 2022-05-12T00:00:00.000Z
+authors: ['josefreitas', 'alexfauquette']
+tags: ['MUI X', 'News']
+---
+
+We’re happy to announce that the Premium plan is [finally out](https://mui.com/pricing/)!
+With it, MUI X officially steps up to the next level, supporting the most advanced use cases for UI components.
+
+The new Premium features are intended to support end-users with the most sophisticated tools to navigate and draw insights from vast amounts of data.
+
+## Row grouping
+
+We started with [row grouping](https://mui.com/x/react-data-grid/row-grouping/), which enables users to group data based on repeating values in the grid.
+For example, users can now group orders by buyers, movies by directors, or cities by the governing parties.
+
+These kinds of use cases would usually be dealt with using a different query on the database, perhaps requiring a new service end-point, maybe even a new front-end, with all the UX challenges this would entail.
+
+Now it’s available at the end-user's fingertip, directly in the data grid.
+Users are encouraged to explore the data and create combinations with multiple grouping criteria,
+and are then able to export these new views to Excel, with the latest Premium feature: Excel export.
+
+## Excel export
+
+One of the most requested features,
+the [“Excel export”](https://mui.com/x/react-data-grid/export/#excel-export) enables users to bring the data from the grid into Excel, while mirroring the existing visualization in the data grid based on filters, sorting, and row grouping.
+
+In the future we intend to support all of the features impacting visualization of the data grid, but we’d love to hear from you if there’s anything in particular you’d like to see in future releases.
+
+## Aggregation (Coming in late Q2)
```

Comment:
```suggestion
## Aggregation (Coming in late CY22 Q2)
```

Reply:
I think it's fine to assume it's going to be this year. We also have it on the public roadmap linked on "what's coming next" section.

Reply:
```suggestion
## Aggregation (Coming in late Q2 this year)
```

Reply:
### Aggregation (coming in late Q2)